qiskit/execute.py

```diff
@@ -86,105 +84,96 @@ def execute(experiments, backend,
             Higher levels generate more optimized circuits,
             at the expense of longer transpilation time.
-                0: No optimization
-                1: Light optimization
-                2: Heavy optimization
-                3: Highest optimization
+            #. No optimization
+            #. Light optimization
+            #. Heavy optimization
+            #. Highest optimization
             If None, level 1 will be chosen as default.
 
-        pass_manager (PassManager):
-            The pass manager to use during transpilation. If this arg is present,
-            auto-selection of pass manager based on the transpile options will be
-            turned off and this pass manager will be used directly.
+        pass_manager (PassManager): The pass manager to use during transpilation. If this
+            arg is present, auto-selection of pass manager based on the transpile options
+            will be turned off and this pass manager will be used directly.
 
-        qobj_id (str):
-            String identifier to annotate the Qobj
+        qobj_id (str): String identifier to annotate the Qobj
 
-        qobj_header (QobjHeader or dict):
-            User input that will be inserted in Qobj header, and will also be
-            copied to the corresponding Result header. Headers do not affect the run.
+        qobj_header (QobjHeader or dict): User input that will be inserted in Qobj header,
+            and will also be copied to the corresponding :class:`qiskit.result.Result`
+            header. Headers do not affect the run.
 
-        shots (int):
-            Number of repetitions of each circuit, for sampling. Default: 1024
+        shots (int): Number of repetitions of each circuit, for sampling. Default: 1024
 
-        memory (bool):
-            If True, per-shot measurement bitstrings are returned as well
+        memory (bool): If True, per-shot measurement bitstrings are returned as well
             (provided the backend supports it). For OpenPulse jobs, only
             measurement level 2 supports this option. Default: False
 
-        max_credits (int):
-            Maximum credits to spend on job. Default: 10
+        max_credits (int): Maximum credits to spend on job. Default: 10
+
+        seed_simulator (int): Random seed to control sampling, for when backend is a simulator
 
-        seed_simulator (int):
-            Random seed to control sampling, for when backend is a simulator
+        default_qubit_los (list): List of default qubit LO frequencies in Hz
 
-        default_qubit_los (list):
-            List of default qubit LO frequencies in Hz
+        default_meas_los (list): List of default meas LO frequencies in Hz
 
-        default_meas_los (list):
-            List of default meas LO frequencies in Hz
+        schedule_los (None or list or dict or LoConfig): Experiment LO
+        configurations, if specified the list is in the format::
```

Comment:
indent here?

Reply:
This is fixed in: https://github.com/Qiskit/qiskit-terra/pull/3902/commits/925d78267bac16443f369b09bf430b54eadbd4f3